docs/self-managed/modeler/web-modeler/configuration.md

```diff
@@ -0,0 +1,120 @@
+| `IDENTITY_BASE_URL`     | [Internal](#notes-on-host-names-and-port-numbers) base URL of the Identity API (used to fetch user data).                 | `http://identity:8080`                                                            |
+
+#### WebSocket
+
+The `webapp` component sends certain events (e.g. "user opened diagram", "user left diagram") to the [WebSocket](#configuration-of-the-websockets-component) server and reacts to events triggered by other users.
+
+| Environment variable      | Description                                                                                                                                   | Example value        | Default value |
+| ------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | -------------------- | ------------- |
+| `PUSHER_HOST`             | [Internal](#notes-on-host-names-and-port-numbers) host name of the WebSocket server.                                                          | `modeler-websockets` | -             |
+| `PUSHER_PORT`             | [Internal](#notes-on-host-names-and-port-numbers) port number of the WebSocket server.                                                        | `8060`               | -             |
+| `PUSHER_APP_ID`           | _must be the same as_ [`PUSHER_APP_ID`](#configuration-of-the-websockets-component)                                                           | `web-modeler`        | -             |
+| `PUSHER_KEY`              | _must be the same as_ [`PUSHER_APP_KEY`](#configuration-of-the-websockets-component)                                                          | \*\*\*               | -             |
+| `PUSHER_SECRET`           | _must be the same as_ [`PUSHER_APP_SECRET`](#configuration-of-the-websockets-component)                                                       | \*\*\*               | -             |
+| `CLIENT_PUSHER_HOST`      | [External](#notes-on-host-names-and-port-numbers) host name on which the Web Modeler client accesses the WebSocket server from the browser.   | `ws.example.com`     | -             |
+| `CLIENT_PUSHER_PORT`      | [External](#notes-on-host-names-and-port-numbers) port number on which the Web Modeler client accesses the WebSocket server from the browser. | `443`                | -             |
+| `CLIENT_PUSHER_KEY`       | _must be the same as_ [`PUSHER_APP_KEY`](#configuration-of-the-websockets-component)                                                          | \*\*\*               | -             |
+| `CLIENT_PUSHER_FORCE_TLS` | Enable TLS encryption for WebSocket connections initiated by the browser.                                                                     | `true`               | `false`       |
+
+### Configuration of the `websockets` component
```

Comment:
Should this be `websocket` rather than `websockets`?

Reply:
The Docker image is called `modeler-websockets`, but I think the spelling here doesn't really matter. Since you've replaced "WebSockets" with "WebSocket" everywhere else, let's also use it without "s" here :).